XenModel/Messages.resx

```diff
@@ -10657,6 +10660,12 @@ Please reconnect the host and try again</value>
   <data name="PROBING_HBA_TITLE" xml:space="preserve">
     <value>Probing for LUNs on {0}</value>
   </data>
+  <data name="PROBLEM_HEALTH_CHECK_SERVICE_DESCRIPTION" xml:space="preserve">

```

Comment:
On the UI "Health Check" is used with capital initial letters as a term, so let's follow this convention here too (and in the messages below). Also, it needs the article.

Reply:
Fixed.